minecraft/protocol/item_stack.go

```diff
@@ -541,6 +543,30 @@ func (c *CraftRecipeOptionalStackRequestAction) Unmarshal(r *Reader) {
 	}
 }
 
+// CraftMineStackRequestAction is an action sent when a player mines a block
+type CraftMineStackRequestAction struct {
+	// Unknown1 ...
+	Unknown1 int32
+	// PredictedDurability ...
+	PredictedDurability int32
+	// StackNetworkID
+	StackNetworkID int32
+}
+
+// Marshal ...
+func (c *CraftMineStackRequestAction) Marshal(w *Writer) {
+	w.Varint32(&c.Unknown1)
+	w.Varint32(&c.PredictedDurability)
+	w.Varint32(&c.StackNetworkID)
+}
+
+// Unmarshal ...
+func (c *CraftMineStackRequestAction) Unmarshal(r *Reader) {
+	r.Varint32(&c.Unknown1)
+	r.Varint32(&c.PredictedDurability)
+	r.Varint32(&c.StackNetworkID)
+}
+
```

Comment:
Changes in this file seem to be redundant.

Reply:
Oops, removed.